recipes/eigen/all/conanfile.py

```diff
@@ -9,10 +9,13 @@ class eigenConan(ConanFile):
                    numerical solvers, and related algorithms."
     license = "MPL-2.0"
     topics = ("eigen", "algebra", "linear-algebra", "vector", "numerical")
-    settings = "os", "compiler", "arch", "build_type"
-    _source_subfolder = "_source_subfolder"
+    settings = "os"
```

Comment:
```suggestion
    settings = "os", "compiler"
```

Keep compiler to avoid problems on Windows

Reply:
What kind of problems?